notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        image_processing = transforms.Compose([
</pre><p> </p><p>Apologies if this comment doesn't make sense. I'm not good at Python.</p><p> </p><p>The comment above says that only postprocess is overridden, but there seems to be some normalization logic here. What is this doing?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>No worries. I updated the comments there.</p>